src/content/components/Tooltip.css

```diff
@@ -67,3 +68,46 @@
   width: 60px;
   color: var(--theme-highlight-gray);
 }
+
+.tooltipTiming {
```

Comment:
I may have rewritten my history incorrectly here. I accidentally squashed my changes on the previous commit.